### DISCUSSION TITLE
Potential opportunity to using the jsregexp package from luarocks by making use of rockspec integration in lazy

### DISCUSSION THREAD

**Discussion by joeljosephreji** [L3MON4D3/LuaSnip#1207] [Ideas]
The new version of lazy.nvim supports rockspec (https://github.com/folke/lazy.nvim?tab=readme-ov-file#%EF%B8%8F-requirements) as mentioned in the requirements tab and it is also something done in the rocks.nvim package manager. Would you be open to including a rockspec file for the jsregexp package so that we could use luarocks here to install the package using this method?

**Comment by L3MON4D3** [comment]
Oh yes please, maybe that simplifies its installation on windows :|

@mrcjkb does/how does the `luarocks-tag-release` workflow interact with a rockspec in the repo?

**Reply by mrcjkb** [reply]
lazy.nvim uses scm rockspecs in the repo root, which typically have a different structure than the release rockspecs generated by luarocks-tag-release.
For example, release rockspecs usually have a source archive or specify a tag.

We have an [open issue to support using scm rockspecs as a template to generate a release rockspec from](https://github.com/nvim-neorocks/luarocks-tag-release/issues/435).

**Reply by L3MON4D3** [reply]
TY!
